### PR TITLE
ticdc: use case name in integration step

### PR DESCRIPTION
**Pull request by amyangfei**
Currently integration test stage name is `integration test step_12`, we can't find the running case directly (have to click the stage name and expand details).

### CODE REVIEW

**Comment by amyangfei** [comment]
ptal @hi-rustin

**Review by hi-rustin — APPROVED** [review]
Great! Thanks!

**Comment by hi-rustin** [comment]
/cc @purelind  
Could you please take a look? Thanks!

**Comment by hi-rustin** [comment]
@purelind  Do we need to test it? Is this a legal name?

**Comment by hi-rustin** [comment]
/hold

I will test it.

File: jenkins/pipelines/ci/ticdc/integration_test_common.groovy [hunk 1]
```diff
@@ -172,7 +172,7 @@ def tests(sink_type, node_label) {
             step_cases.add(case_names)
         }
         step_cases.eachWithIndex { case_names, index ->
-            def step_name = "step_${index}"
+            def step_name = "step_${index} case " + case_names.join(" ")
```
Comment (amyangfei) [inline jenkins/pipelines/ci/ticdc/integration_test_common.groovy:175]:
```suggestion
            def step_name = "step_${index} " + case_names.join(" ")
```
to make `step_name` shorter

**Comment by purelind** [comment]
> /hold
> 
> I will test it.

Let me test this later

File: jenkins/pipelines/ci/ticdc/integration_test_common.groovy [hunk 2]
```diff
@@ -172,8 +172,8 @@ def tests(sink_type, node_label) {
             step_cases.add(case_names)
         }
         step_cases.eachWithIndex { case_names, index ->
-            def step_name = "step_${index}"
-            test_cases["integration test ${step_name}"] = {
+            def step_name = "${index} ${case_names.join(" ")}"
```
Comment (amyangfei) [inline jenkins/pipelines/ci/ticdc/integration_test_common.groovy:175]:
```suggestion
            def step_name = case_names.join(" ")
```

File: jenkins/pipelines/ci/ticdc/integration_test_common.groovy [hunk 3]
```diff
@@ -172,8 +172,8 @@ def tests(sink_type, node_label) {
...
+            test_cases["${step_name}"] = {
```
Comment (amyangfei) [inline jenkins/pipelines/ci/ticdc/integration_test_common.groovy:176]:
```suggestion
            test_cases[step_name] = {
```

**Comment by hi-rustin** [comment]
@purelind  Already tested. Could you please help merge this PR?

Maybe the code can still be optimized and I will follow up to optimize it. Let's get it running first!

**Review by hi-rustin — APPROVED** [review]

**Review by purelind — APPROVED** [review]
LGTM

**Pull request merged**